lib/iris/plot.py

```diff
@@ -610,14 +617,19 @@ def contour(cube, *args, **kwargs):
 
     Kwargs:
 
-    * coords: list of :class:`~iris.coords.Coord` objects or coordinate names
-        Use the given coordinates as the axes for the plot. The order of the
-        given coordinates indicates which axis to use for each, where the first
-        element is the horizontal axis of the plot and the second element is
-        the vertical axis of the plot.
+    * coords: list of :class:`~iris.coords.Coord` objects or
+        coordinate names Use the given coordinates as the axes for the
+        plot. The order of the given coordinates indicates which axis
+        to use for each, where the first element is the horizontal
+        axis of the plot and the second element is the vertical axis
+        of the plot.
+
+    * axes: :class:`cartopy.mpl.geoaxes.GeoAxes` instance to call the
```

Comment:
As you say, as long as I raise an exception when it's expecting a GeoAxes I think we can safely change all these to regular matplotlib axes.